src/modules/layout/containers/HeaderContainer.js

```diff
@@ -2,6 +2,23 @@
 
 import Header from '../components/Header'
 import { withTheme } from 'styled-components/native'
+import type { StateType } from '../../app/StateType'
+import { connect } from 'react-redux'
+import React from 'react'
+import { withNavigation } from 'react-navigation'
 
-// $FlowFixMe
-export default withTheme(Header)
+const mapStateToProps = (state: StateType, ownProps) => {
+  const key: string = ownProps.navigation.getParam('key')
```

Comment:
This is the `key` from the navigation actions which lead to the dashboard/categories or events, right?

Reply:
Probably, yes. Should be the key of the current screen